pkg/frontend/admin_platformworkloadidentityroleset_put_test.go

```diff
@@ -0,0 +1,790 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/admin"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestPlatformWorkloadIdentityRoleSetPut(t *testing.T) {
+	ctx := context.Background()
+
+	type test struct {
+		name           string
+		fixture        func(f *testdatabase.Fixture)
+		body           *admin.PlatformWorkloadIdentityRoleSet
+		wantStatusCode int
+		wantResponse   *admin.PlatformWorkloadIdentityRoleSet
+		wantError      string
+		wantDocuments  []*api.PlatformWorkloadIdentityRoleSetDocument
+	}
+
+	for _, tt := range []*test{
+		{
+			name: "updating known version",
```

Comment:
nit: would again be good to get a full sentence as the name - updating (I would probably rename this to `putting` or `PUT on known version` or similar for consistency) a known version is doing what exactly? Same for a few other tests in this file.